app/assets/stylesheets/plugins/wice_grid.scss

```diff
@@ -1,148 +1,81 @@
+
+    .expand-multi-select-icon, .collapse-multi-select-icon{
+      width: 10px;
       height: 10px;
+      display: inline-block;
       margin-left: 5px;
       vertical-align: top;
-      width: 10px;
-    }
-
-    .collapse-multi-select-icon {
-      background: transparent asset-url("/assets/icons/grid/collapse.gif") no-repeat;
-    }
-
-    .expand-multi-select-icon {
-      background: transparent asset-url("/assets/icons/grid/expand.gif") no-repeat;
     }
   }
 }
 
-.wice-grid {
-  .desc,
-  .asc {
-    padding-right: 18px;
-    text-decoration: none;
-  }
-
-  .desc {
-    background: transparent asset-url("/assets/icons/grid/arrow_down.gif") right no-repeati;
-  }
-
-  .asc {
-    background: transparent asset-url("/assets/icons/grid/arrow_up.gif") right no-repeat;
-  }
+.wice-grid{
 
-  .submit.clickable {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/table_refresh.png") no-repeat;
-  }
-
-  .reset.clickable {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/table.png") no-repeat;
-  }
-
-  .wg-show-filter.clickable,
-  .wg-hide-filter.clickable {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/page_white_find.png") no-repeat;
-  }
-
-  .export-to-csv-button.clickable {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/page_white_excel.png") no-repeat;
-  }
-
-  .clickable.select-all {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/tick_all.png") no-repeat;
-    float: left;
+  .desc,  .asc{
+    padding-right: 18px;
+    text-decoration:none;
   }
 
-  .clickable.deselect-all {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/untick_all.png") no-repeat;
-    float: left;
+  .clickable.select-all, .clickable.deselect-all{
+    float:left;
   }
 
-  // in case of twitter bootstrap :)
-  thead th select {
+  /* in case of twitter bootstrap :) */
+  thead th select{
     width: auto;
   }
 
-  .pagination {
+  .pagination{
+    margin: 0px;
     float: left;
-    margin: 0;
   }
 
-  tr {
-    .wg-filter-row input[type=text] {
-      width: 100px;
-    }
+  tr.wg-filter-row  input[type=text] {
+    width: 100px;
   }
 
-  .pagination-status {
-    float: right;
+  .pagination_status {
     font-weight: bold;
+    float: right;
   }
 }
 
-.wice-grid-query-panel {
-  li {
-    list-style-type: none;
-  }
-
-  ul {
-    margin-left: 0;
-  }
-
-  a {
-    .wice-grid-delete-query .delete-icon {
+.wice-grid-query-panel{
+  li {list-style-type: none; }
 
-      @include icon-dimensions;
-      background: transparent asset-url("/assets/icons/grid/delete.png") no-repeat;
-      float: left;
-    }
-  }
+  ul {margin-left: 0 }
 }
 
-input {
-  .wice-grid-save-query-field {
-    display: inline-block;
-    margin-right: 10px;
-    width: auto;
-  }
+input.wice-grid-save-query-field{
```

Comment:
Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space